gpiozero/output_devices.py

```diff
@@ -797,20 +797,31 @@ class Motor(SourceMixin, CompositeDevice):
         The GPIO pin that the backward input of the motor driver chip is
         connected to.
 
+    :param int enable:
+        (Optional) The GPIO pin that enables the motor. Required for some motor
+        controller boards. Defaults to ``None``.
+
     :param bool pwm:
         If ``True`` (the default), construct :class:`PWMOutputDevice`
         instances for the motor controller pins, allowing both direction and
         variable speed control. If ``False``, construct
         :class:`DigitalOutputDevice` instances, allowing only direction
         control.
     """
-    def __init__(self, forward=None, backward=None, pwm=True):
+    def __init__(self, forward=None, backward=None, enable=None, pwm=True):
```

Comment:
To maintain backwards-compatibility, shouldn't the `enable` arg be added *after* the `pwm` arg?